world.js

```diff
@@ -583,70 +589,19 @@ world.getNpcFromPhysicsId = physicsId => {
   return null;
 };
 
-let animationMediaStream = null
-let networkMediaStream = null;
-const _latchMediaStream = async () => {
-  networkMediaStream = await navigator.mediaDevices.getUserMedia({
-    audio: true,
-  });
-  if (channelConnection) {
-    await channelConnection.setMicrophoneMediaStream(networkMediaStream);
-  }
-};
-const _unlatchMediaStream = async () => {
-  if (channelConnection) {
-    await channelConnection.setMicrophoneMediaStream(null);
-  }
-
-  const tracks = networkMediaStream.getTracks();
-  for (const track of tracks) {
-    track.stop();
-  }
-  networkMediaStream = null;
-};
-
 const micButton = document.getElementById('key-t');
+
 world.toggleMic = async () => {
-  if (!animationMediaStream) {
+  if (!wsrtc.mediaStream) {
```

Comment:
Maybe better to make this a method?

Reply:
```if (this.mediaStream) {
      throw new Error('mic already enabled');
    }```
    this seems to be the check wsrtc uses internally.

Reply:
> this seems to be the check wsrtc uses internally.  

Yes, but `mediaStream` should be a private variable (to that library).